spec/person_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+describe Person do
+  let(:person) { Person.new(20, 'Alice', parent_permission: false) }
+
+  describe '#initialize' do
+    it 'creates a new Person object with the given age' do
+      expect(person.age).to eq(20)
+    end
+
+    it 'creates a new Person object with the given name' do
+      expect(person.name).to eq('Alice')
+    end
+
+    it 'creates a new Person object with parent permission' do
+      expect(person).to have_attributes(parent_permission: false)
+    end
+
+    it 'creates a new Person object with an ID between 1 and 1000' do
+      expect(person.id).to be_between(1, 1000).inclusive
+    end
+
+    it 'creates a new Person object with an empty rentals list' do
+      expect(person.rentals).to be_empty
+    end
+  end
+
+  describe '#add_rentals' do
+    it 'adds a rental to the person\'s list of rentals' do
+      rental = double('rental')
+      person.add_rental(rental)
+      expect(person.rentals).to include(rental)
+    end
+  end
+
+  describe '#correct_name' do
+    it 'returns the person\'s name' do
+      expect(person.correct_name).to eq('Alice')
+    end
+  end
+
+  describe '#can_use_services?' do
+    it 'returns true if the person is 18 or older' do
+      expect(person.can_use_services?).to be_truthy
+    end
+
+    it 'returns true if the person is under 18 and has parent permission' do
+      person = Person.new(17, parent_permission: true)
+      expect(person.can_use_services?).to be_truthy
+    end
+
+    it 'returns false if the person is under 18 and does not have parent permission' do
+      person = Person.new(17, parent_permission: false)
+      expect(person.can_use_services?).to be_falsey
+    end
+  end
+end
```

Comment:
- [ ] Great work writing these tests. Based on the requirements, you ought to have unit tests for all methods in a class. Kindly add unit tests for the following methods in the `Person` class:
- `add_rental`
- `correct_name` 💻

Reply:
Hello @Gambit142 .
Thank you for the review. I believe we already included tests for both methods in the highlighted test. Specifically [line 28](https://github.com/eric2umeh/school-library-oop-ruby/blob/967def41dea2216de945870662e5bae8f428fcb5/spec/person_spec.rb#L28) for the 'add_rental' method, and [line 34](https://github.com/eric2umeh/school-library-oop-ruby/blob/967def41dea2216de945870662e5bae8f428fcb5/spec/person_spec.rb#L36).  Could you please clarify if they still require modification.

Reply:
Hi @Thoth1111, You are right, it was an oversight. Please send it for another review. It would be approved. I am sorry for the oversight and inconvenience. 👍 😎

Reply:
Alright, no worries. Thank you.